p2p/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name       = "p2p"
+version    = "0.1.0"
+edition    = "2018"
+license    = "Apache-2.0"
+repository = "https://github.com/informalsystems/tendermint-rs"
+readme     = "README.md"
+keywords   = ["p2p", "tendermint", "cosmos"]
+authors    = [
+  "Tony Arcieri <tony@iqlusion.io>",
+  "Ismail Khoffi <Ismail.Khoffi@gmail.com>"
+]
+
+description = """
+    The Tendermint P2P stack.
+    """
+
+[dependencies]
+chacha20poly1305 = "0.7"
+ed25519-dalek = "1"
+eyre = "0.6"
+hkdf = "0.10.0"
+merlin = "2"
+prost = "0.6"
+prost-amino = "0.6"
+prost-amino-derive = "0.6"
+prost-derive = "0.6"
+rand_core = { version = "0.5", features = ["std"] }
+sha2 = "0.9"
+subtle = "2"
+subtle-encoding = { version = "0.5", features = ["bech32-preview"] }
+tendermint = { version = "=0.17.0-rc3", features = ["secp256k1"] }
+tendermint-proto = "=0.17.0-rc3"
```

Comment:
Dependencies on crate in the same repository should be local relative `../<name>` and ideally bundled together for easier visibility.

Reply:
Should I use multiple locations https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations so crate can be published on crates.io? otherwise it may not be possible: "Note: crates.io does not allow packages to be published with path dependencies (path dev-dependencies are ignored)."

Reply:
Good call actually, will tmkms depend on the crates.io version? @tony-iqlusion

Reply:
You can include a path dependency so long as there's also a version number.

TMKMS will need a crates.io release, yes.